redis/src/cluster_topology.rs

```diff
@@ -69,6 +69,7 @@ pub fn get_slot(key: &[u8]) -> u16 {
 pub(crate) fn parse_and_count_slots(
     raw_slot_resp: &Value,
     tls: Option<TlsMode>,
+    calling_host: &str,
```

Comment:
calling_host might not be the best name as it can be confused with the client host